contrib/dockerd-rootless.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e -x
+if ! [ -w $XDG_RUNTIME_DIR ]; then
+	echo "XDG_RUNTIME_DIR needs to be set and writable"
+	exit 1
+fi
+if ! [ -w $HOME ]; then
```

Comment:
Do we still need `$HOME` if `$XDG_DATA_HOME` is set?

Reply:
Probably no, but it would not hurt to check?